### DISCUSSION TITLE
Themes for Generated HTML

### DISCUSSION THREAD

**Discussion by RoyiAvital** [facelessuser/MarkdownPreview#128] [General]
The basic CSS Theme is really great.

But it misses some features (Like Admonitions).
I was wondering, where can I get themes which are compatible with the defaults?

I really like, for instance, the theme in [documentation site](https://facelessuser.github.io/MarkdownPreview):

![image](https://user-images.githubusercontent.com/7347975/119226548-56808200-bb12-11eb-88c8-b0d74aac41b3.png)

Is there a CSS file of I can just include using [Override CSS by File Type](https://facelessuser.github.io/MarkdownPreview/usage/#override-css-by-file-type)?

**Comment by facelessuser** [comment]
> The basic CSS Theme is really great.
>
> But it misses some features (Like Admonitions).

I'm confused by this statement. I can use Admonitions just fine when using the `markdown` parser (not `github`).

```
!!! caution "Admonition"
    Working just fine!
```

<img width="959" alt="Screen Shot 2021-05-22 at 7 58 03 AM" src="https://user-images.githubusercontent.com/1055125/119229101-8f4d3680-bad3-11eb-97c5-19609da0d002.png">

**Reply by RoyiAvital** [reply]
This is what I get:

![image](https://user-images.githubusercontent.com/7347975/119229418-ac5c2680-bb20-11eb-9c88-b10ba8f20566.png)

![image](https://user-images.githubusercontent.com/7347975/119229443-ceee3f80-bb20-11eb-869e-dd2342cf0b03.png)

Could it be Sublime 4 issue?
As I'm also having issues with `b64` from your extensions package.

**Reply by facelessuser** [reply]
I'm on Sublime 4. I guess I need more information. I don't understand how you are getting numbered headers either. Are you really using the parser you think you are using? Can you give me a step-by-step so I can try and reproduce it?

**Reply by RoyiAvital** [reply]
OK,

I think I created a minimal working example:

[AdoTest.zip](https://github.com/facelessuser/MarkdownPreview/files/6526582/AdoTest.zip)

It includes the HTML I get.


My `MarkdownPreview.sublime-settings`:

```
{
    "js": {
        "markdown": [
            "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js",
            "res://MarkdownPreview/js/math_config.js"
        ]
    },
    
    "markdown_extensions": [
    	{
        	"pymdownx.arithmatex":
        	{
            	"generic": true,
                "smart_dollar": false
            }
        },
    ],

    "strip_yaml_front_matter": true,
}
```

Any more information I could add?

**Reply by facelessuser** [reply]
Nope, that is fine. You've overridden all the extensions and have not added admonitions, only arithmatex.

**Reply by RoyiAvital** [reply]
I thought the user settings file is appended but I guess it is overriding, right?

**Reply by facelessuser** [reply]
It has never used appending as there would be no way to remove extensions you are not interested in.

**Reply by RoyiAvital** [reply]
OK, Could you check the `b64` thing at the attached file?

In that file I enabled `b64` in `YAML`.

**Reply by facelessuser** [reply]
I'll try when I have a chance.

**Reply by RoyiAvital** [reply]
I created a better example for the `b64` issue in the Issues section.

Do you have ideas where to get more CSS which fit reports and stuff like that?

Thank You.

**Reply by facelessuser** [reply]
> I created a better example for the `b64` issue in the Issues section.

Yeah, I saw, and I'll take a look. As far as I know, things are working just fine with B64, but I will confirm.

> Do you have ideas where to get more CSS which fit reports and stuff like that?

I'm not fully understanding this question here, what do you mean by "fit reports and stuff like that"?

**Reply by RoyiAvital** [reply]
For technical writings.

I really like the theming you created for your documentation site, how can I have it as a theme?
